.github/workflows/mypy_primer_comment.yml

```diff
@@ -0,0 +1,74 @@
+name: Run mypy_primer
+
+on:
+  workflow_run:
+    workflows:
+      - Run mypy_primer
+    types:
+      - completed
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  comment:
+    name: Comment PR from mypy_primer
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download diffs
+        uses: actions/github-script@v3
+        with:
+          script: |
+            const fs = require('fs');
+            const artifacts = await github.actions.listWorkflowRunArtifacts({
+               owner: context.repo.owner,
+               repo: context.repo.repo,
+               run_id: ${{ github.event.workflow_run.id }},
+            });
+            const matchArtifact = artifacts.data.artifacts.filter((artifact) =>
+              artifact.name == "mypy_primer_diffs")[0];
```

Comment:
Can there be multiple matches? There was code to sort something by date, but I'm not sure if it's still needed, since this uses `run_id`.

Reply:
We could probably just do `artifacts.data.artifacts[0]`, although this is a bit safer, I guess.